### ISSUE TITLE
UI - Indicate profile status - search results, browse results

### ISSUE THREAD

**Issue by RobinaSanderson** [AtlasOfLivingAustralia/profile-hub#481]
As a user of the eFlora search and browse
I want to know what profiles in results are stubs
So that I don't need to open them to know they don't currently have content

Question: 
- If this is a simple boolean flag/icon, do we consider legacy and in review profiles as also complete for this purpose, and empty and partial as incomplete? Partial could be considered still useful?
- Otherwise, we could display the actual status appended to the name? 

This issue relates to #480 and #477

**Comment by RobinaSanderson** [comment]
@ChrisPalmer1 - Is this required for Production release?

**Comment by ChrisPalmer1** [comment]
Yes, would be good for production release; sorry we left it without a comment. It would be good to have a solution in place before we roll the eFlora out to botanical contributors.
Thanks, Chris

**Comment by m-hope** [comment]
changed priority to High as it is a precursor to #483

**Comment by ben3000** [comment]
What is the technical definition of a stub profile? Is it any profile that has certain empty properties? If so, which properties? Or is it something the profile author can define irrespective of a profile's actual content?

**Comment by RobinaSanderson** [comment]
Hi @ben3000 a stub profile is a profile with STATUS_EMPTY.
Basically, where the profile has been created from a listing in the master list, but no other fields have been populated. I think collections without a master list will not have stub profiles, as the profiles will need to be created by a data import (Legacy), or by someone creating a profile (Partial).
@m-hope Do you have a different opinion?

**Comment by m-hope** [comment]
Hi @ben3000 , @RobinaSanderson is correct.  Stub profile == profile with STATUS_EMPTY

The original idea behind the stub profile was if a masterlist contained a name that did not have a matching profile in existence, then the system would create a blank 'stub' on the fly that would display an 'under construction' message... we've found that this isn't possible and now at the time a masterlist is loaded, empty profiles (with status set to STATUS_EMPTY) will be created in the database for any name in the list that doesn't already have a profile.

**Comment by RobinaSanderson** [comment]
Agreement this is not needed for the search results, need to decide if this is needed on browse results.
@KevinThiele @ChrisPalmer1 @m-hope Can this be put on the backlog for the browse results?

**Comment by KevinThiele** [comment]
Yes I think so - browse as for search.
